setup.py

```diff
@@ -149,9 +149,9 @@ def customize_compiler(compiler):
                 flags.append('-mmacosx-version-min=10.7',)
                 if tuple(
                    map(int, platform.mac_ver()[0].split('.'))
-                ) >= (10, 9):
+                   ) >= (10, 9):
```

Comment:
I think the correct response to this lint is to indent the `map` line by one more space instead of the paren

Reply:
Or save width a simpler and cleaner way:
```python
                  macver = tuple(map(int, platform.mac_ver()[0].split('.')))
                  if macver >= (10, 9):
```